ocean_lib/models/data_token.py

```diff
@@ -321,13 +319,13 @@ def get_event_logs(
         event = getattr(self.events, event_name)
         filter_params = filter_args or {}
         event_filter = EventFilter(
```

Comment:
Since this event_filter is never reused for polling, I recommend replacing it with `event().getLogs()` like so:

```python
logs = event().getLogs(
    argument_filters = filter_params, 
    from_block = from_block,
     to_block = to_block,
)
```